dotCMS/src/main/java/com/dotmarketing/business/PermissionBitFactoryImpl.java

```diff
@@ -96,18 +112,8 @@
 	 * 2. Reference id
 	 * 3. Type
 	 */
-//	private final String insertPermissionReferenceSQL =
-//		(DbConnectionFactory.getDBType().equals(DbConnectionFactory.MYSQL) || DbConnectionFactory.getDBType().equals(DbConnectionFactory.MSSQL)?
-//		"insert into permission_reference (asset_id, reference_id, permission_type) " +
-//		"	SELECT ?,?,? FROM permission_reference ":
-//		DbConnectionFactory.getDBType().equals(DbConnectionFactory.ORACLE)?
-//		"insert into permission_reference (id, asset_id, reference_id, permission_type) " +
-//		"	SELECT (SELECT permission_reference_seq.NEXTVAL), ?, ?, ? FROM permission_reference ":
-//		"insert into permission_reference (id, asset_id, reference_id, permission_type) " +
-//		"	SELECT (SELECT nextval('permission_reference_seq')), ?, ?, ? FROM permission_reference ") +
-//		" WHERE not exists (SELECT asset_id from permission_reference where asset_id = ?)";
-
-	private final String insertPermissionReferenceSQL =
+
+	private static final String insertPermissionReferenceSQL =
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)